samples/led-matrix-weather/Program.cs

```diff
@@ -0,0 +1,274 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Iot.Device.LEDMatrix;
+using Iot.Device.Graphics;
+using System.Drawing;
+using System.Numerics;
+using System.Diagnostics;
+using System.Net;
+using System.Net.Sockets;
+using System.Net.NetworkInformation;
+
+namespace LedMatrixWeather
+{
+    partial class Program
+    {
+        static Action<RGBLedMatrix> s_scenario = WeatherDemo;
+        static Stopwatch s_showLocalIp = null;
+        static string[] s_ips;
+        static bool s_networkAvailable = false;
+        static Weather s_client;
+        static OpenWeatherResponse s_weatherResponse;
+        static TimeZoneInfo s_timeZonePst = TimeZoneInfo.FindSystemTimeZoneById("America/Los_Angeles");
+
+        static void Main(string[] args)
+        {
+            if (args.Length != 1)

```

Comment:
Shouldn't we just show example data in case no args ar passed in? that seems to be what is suggested by your Usage text. I would just add support for a `/?` or `-?` that would print the Usage.

Reply:
I'll prefer to refrain from adding command line parser here. Once I need more options will use library for that.

Showing example data was meant to be a fallback option since this likely will be used in headless scenario and better to show something than nothing.